### DISCUSSION TITLE
丽都美食家：浅谈芬达(1)

### DISCUSSION THREAD

**Discussion by sb35386057** [share121/inter-knot#5435] [灌水]
![IMG_20240927_220839](https://github.com/user-attachments/assets/ccca4e07-2269-441a-a3e3-571d6808f14a)
本次讨论内容为前5个口味，带有强烈主观个人意愿。
经常喝饮料的朋友们都知道，芬达这口味还不少，这周连续喝了几种不同口味的芬达，接下来来谈谈我个人的见解：
源味芬达：原味就是原味，其他口味是超不过的
水蜜桃味：相当于糖水蜂蜜水喝着，还行，一般般吧
青苹果味：果汁味已经挺浓了，能达到喝起来知道是青苹果味儿
葡萄味：甜的，加气泡的刺激性，想假装喝葡萄酒的朋友们可以拿这个假装装逼
蓝色口味：名字不知道了，喝起来主要是甜味儿，再加上一点什么植物味，脑子里感觉有点异味，可能也没有，或许是看到蓝色联想到洁厕灵了吧。
大家觉得以上口味怎么样呢，在评论区发表你的见解，以及有没有朋友可以把后面三个口味的味道发出来，客观讨论一下。

**Comment by KELERSAMA** [comment]
给我喝seven up 
<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Reply by sb35386057** [reply]
芬达对标不是美年达吗，七喜对标雪碧吧
![Uploading 68747470733a2f2f696d672d7374617469632e6d69686f796f2e636f6d2f636f6d6d756e6974797765622f75706c6f61642f39323439626433363132356239353364646434343562313931353134323764362e706e67.png…]()

**Comment by chenyunji** [comment]
我巨喜欢苹果味的芬达，就是卖的没有别的饮料那么多

**Reply by sb35386057** [reply]
苹果味儿不错的，喝起来就知道是苹果味儿

**Comment by sb35386057** [comment]
西瓜味的芬达没喝过,但同喝过同口味的美年达，说实话喝不出来西瓜味儿，因为西瓜本来就是水甜，总之喝起来是甜就是了，最后两个口味是什么来着

**Comment by hpp326** [comment]
国内都没见过，没想到伦敦这边的街边便利店居然有卖，还挺全。打算试试
![IMG_20240915_193002](https://github.com/user-attachments/assets/bfe397db-763a-4856-90cb-baa3b1c18023)

**Comment by hpp326** [comment]
![Uploading IMG_20250123_172714.jpg…]()
桃子味的好喝耶

**Comment by BSJ3456A** [comment]
个人甜食爱好者喜欢喝青苹果和西瓜味的 甜甜的都很好喝 西瓜味带点清凉蛮适合夏天喝的૮₍◜ෆ◝.₎ა
